### DISCUSSION TITLE
Celebrate the birth of the first tool program ready for use in the production environment

### DISCUSSION THREAD

**Discussion by shove70** [vlang/v#15617] [General]
<img width="858" alt="image" src="https://user-images.githubusercontent.com/27794478/187667876-30e7948d-4032-4f2e-8c03-6b99f77e6d61.png">

After seven or eight days, the first small tool written in V was finally born. After testing, it runs well and has good performance.

Although there were some obstacles in the whole process, the overall feeling was smooth. During this period, I submitted several issues and PR, which were strongly supported and encouraged by experts in the community. Finally, I felt that everything was perfect. Thank you for your efforts to provide such a good V. I love you.

**Comment by dumblob** [comment]
Congratulations! That is good news. Feel free to chime in https://github.com/vlang/v/discussions/14222 and describe your experience a little more technically :wink:.

Also is the SW an [ECS-inspired architecture](https://github.com/vlang/ui/issues/7#issuecomment-855729677 )?

Could you share (parts of) the code?
